### DISCUSSION TITLE
CA cert configuration of LDAP_CA_CERT_PATH

### DISCUSSION THREAD

**Discussion by dcxhann** [danny-avila/LibreChat#3498] [Troubleshooting]
### What happened?

Hi, there,

Want to preface great work on this project! Anyway I followed the tutorial guide and got LDAP working, but did encounter the following issue with specifying LDAP_CA_CERT_PATH. I was able to resolve it, see below for resolution.

Basically I noticed LDAP_CA_CERT_PATH parameter will only work if I uploaded the LDAP server's CA cert into the container image using "docker cp certfile <containerID>:/path/to/certile" command. If this is not performed then librechat would fail to start with the following error:

Error: ENOENT: no such file or directory, open '/path/to/certfile'

Hoping librechat can be updated so that it will use the CA cert specified on the config instead of needing to upload the cert onto the container image.

### Steps to Reproduce

step 1. Configure librechat with LDAP config and specify a cert in LDAP_CA_CERT_PATH parameter
step 2. start librechat via "docker-compose up"

### What browsers are you seeing the problem on?

Firefox

### Relevant log output

```shell
Error: ENOENT: no such file or directory, open '/path/to/certfile'
    at Object.openSync (node:fs:573:18)
    at Object.readFileSync (node:fs:452:35)
    at /app/api/strategies/ldapStrategy.js:57:24
    at Object.<anonymous> (/app/api/strategies/ldapStrategy.js:62:11)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/path/to/certfile'
}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I think it's a matter of editing/creating an override file for use with docker:

https://www.librechat.ai/docs/configuration/docker_override

This way, it mounts the file to the container on startup, no additional steps required
